cmd/auth/describe.go

```diff
@@ -0,0 +1,173 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+var authTemplate = fmt.Sprintf(`{{"Workspace:" | bold}} {{.Details.Host}}.
+{{if .Username }}{{"User:" | bold}} {{.Username}}.{{end}}
+{{"Authenticated with:" | bold}} {{.Details.AuthType}}
+{{if .AccountID }}{{"Account ID:" | bold}} {{.AccountID}}.{{end}}
+-----
+%s
+`, configurationTemplate)
+
+var errorTemplate = fmt.Sprintf(`Unable to authenticate: {{.Error}}
+-----
+%s
+`, configurationTemplate)
+
+const configurationTemplate = `Configuration:
+  {{- $details := .Details}}
+  {{- range $k, $v := $details.Configuration }}
+  {{if $v.AuthTypeMismatch}}x{{else}}✓{{end}} {{$k | bold}}: {{$v.Value}}
+  {{- if $v.Source }}
+  {{- " (from" | italic}} {{$v.Source.String | italic}}
+  {{- if $v.AuthTypeMismatch}}, {{ "not used for auth type " | red | italic }}{{$details.AuthType | red | italic}}{{end}})
+  {{- end}}
+  {{- end}}
+`
+
+func newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describes which auth credentials is being used by the CLI and identify their source",
+	}
+
+	var showSensitive bool
+	var isAccount bool
+	cmd.Flags().BoolVar(&showSensitive, "sensitive", false, "Include sensitive fields like passwords and tokens in the output")
+	cmd.Flags().BoolVar(&isAccount, "account", false, "Describe the account client instead of the workspace client")
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+		var status *authStatus
+		var err error
+		if isAccount {
+			status, err = getAccountAuthStatus(cmd, args, showSensitive, func(cmd *cobra.Command, args []string) (*config.Config, error) {
+				err := root.MustAccountClient(cmd, args)
+				return root.ConfigUsed(cmd.Context()), err
+			})
+		} else {
+			status, err = getWorkspaceAuthStatus(cmd, args, showSensitive, func(cmd *cobra.Command, args []string) (*config.Config, error) {
```

Comment:
When I run this I get the `DEFAULT` profile details, but the `profile` isn't included in the configuration output.

I was puzzled where the other configs came from and had to puzzle a bit to figure out it was the `DEFAULT` profile.